File: SPOJ/Dobradura.cpp
#include <stdio.h>

int n, t = 1;

int main(void) {
    while(scanf("%d", &n) && n != -1) {
        printf("Teste %d\n%d\n", t, (1 + (1 << n))*(1 + (1 << n))); t++;
    }
    return 0;
}

File: SPOJ/Dobradura.lua
teste = 1
while true do
	n = tonumber(io.read())
	if n == -1 then
		break
	else
		print("Teste " .. teste .. "\n" .. (1 + 2^n)*(1 + 2^n))
		teste = teste + 1
	end
end
